apps/web/app/not-found.tsx

```diff
@@ -22,9 +22,11 @@ export const generateMetadata = async () => {
 const ServerPage = async () => {
   const h = await headers();
   const nonce = h.get("x-nonce") ?? undefined;
+  const host = h.get("x-forwarded-host") ?? "";
```

Comment:
<img width="547" alt="Screenshot 2025-04-22 at 12 43 02 PM" src="https://github.com/user-attachments/assets/badf5635-42dd-4cae-a4f6-99fcd96e482f" />

This is what we relied on before I turned NotFound into a client component

so it should be safe